instrumentation/logback/logback-mdc-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/mdc/v1_0/OpenTelemetryAppender.java

```diff
@@ -44,9 +44,36 @@ public void setAddBaggage(boolean addBaggage) {
     this.addBaggage = addBaggage;
   }
 
+  /**
+   * {@code <traceIdKey>trace_id</traceIdKey>}
```

Comment:
![image](https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/88769332/67fed3b3-4259-4b46-b4af-f90aa4424c8e)
Is it changed like this, because I wrote it based on addBagge before